packages/instrument-utils/src/storedInstrument/StoreInstrumentFieldset/StoreInstrumentFieldset.tsx

```diff
@@ -23,17 +23,10 @@ interface WithStorageSettings {
 }
 
 const useProps = (
-    context: CheckoutContextProps,
     props: StoreInstrumentFieldsetProps,
     paymentForm: PaymentFormService,
 ): WithStorageSettings => {
-    const {
-        checkoutState: {
-            data: { getInstruments },
-        },
-    } = context;
-
-    const allInstruments = getInstruments();
+    const allInstruments = props.instruments;
```

Comment:
It might be nicer to get `instruments` from `props` on line 33 and update `allInstruments` with `instruments`.